### DISCUSSION TITLE
discogs_albumid is always 0

### DISCUSSION THREAD

**Discussion by jordanbs** [beetbox/beets#4224] [Q&A] [answered]
Most of my music is imported using the discogs plugin (beets v1.6.0, formerly imported via beets 1.4.9), but the every single `discogs_albumid` field is equal to 0. I do notice, however, the `mb_albumid` field contains what should be in discogs_albumid.

E.g.
```
         data_source: Discogs
     discogs_albumid: 0
    discogs_artistid: 1954908
     discogs_labelid: 1085864
    mb_albumartistid: 1954908
          mb_albumid: 10346192
   mb_releasegroupid: 1185315
```
Is this expected?

**Comment by sampsyo** [comment, marked as answer]
Nope; looks like a problem! It is probably related to #4080. I think #4085 probably should have eliminated `extract_release_id` altogether and replaced it with the new `extract_release_id_regex` or similar. Would you mind opening an issue? We should probably consider this a bug.

**Reply by jordanbs** [reply]
Filed!
https://github.com/beetbox/beets/issues/4225
